corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views.js

```diff
@@ -576,9 +584,9 @@ hqDefine("cloudcare/js/formplayer/menus/views", function () {
                     L.marker(homeLatLng, { icon: homeLocationIcon })
                         .bindPopup(gettext("Your location"))
                         .addTo(addressMap);
-                    latLons.push(homeLatLng);
+                    latLngs.push(homeLatLng);
```

Comment:
Slight preference for names that are either real words or commonly used abbreviations, although there isn't a really obvious one here. Maybe `coords`?